spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/MessageConverterConfigurer.java

```diff
@@ -83,25 +88,35 @@
 	private final BindingServiceProperties bindingServiceProperties;
 
 	private ConfigurableListableBeanFactory beanFactory;
+	
+	private final Map<String, PartitionKeyExtractorStrategy> partitionKeyExtractors;
```

Comment:
Need to configure STS to trim trailing blanks - I use the anyEdit plugin.

`git log -p --check` ...

<img width="1366" alt="screen shot 2018-01-16 at 2 09 00 pm" src="https://user-images.githubusercontent.com/483832/35007212-e84a9366-fac6-11e7-9e13-8540e1d48ea4.png">